buildSrc/src/main/kotlin/org/sonarsource/kotlin/buildsrc/tasks/CreateRuleStubsTask.kt

```diff
@@ -22,6 +16,8 @@ import kotlin.io.path.readText
 import kotlin.io.path.writeLines
 import kotlin.io.path.writeText
 
+private const val KOTLIN_GRADLE_BASEDIR = "sonar-kotlin-gradle"
```

Comment:
This is not used everywhere where it could. It is also specific to the Gradle stubs only, it should be moved in the Gradle-specific file along with the Gradle-specific logic (see file-level comment above).

Reply:
Done.